### PR TITLE
Add Yale KeylessConnected-plus update Connexis l1 help and formatting

### PR DESCRIPTION
**Pull request by matthewcky2k**
Added YD-01-CON https://products.z-wavealliance.org/products/2576

### CODE REVIEW

File: config/assa_abloy/KeylessConnected-plus.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		</Value>
+		<Value type="list" size="1" genre="config" instance="1" index="2" label="Auto Relock" value="0">
+			<Help></Help>
+			<Item label="On" value="255" />
+			<Item label="Off" value="0" />
+		</Value>
+		<Value type="byte" genre="config" instance="1" index="3" label="Auto-re-lock Time" size="1" min="7" max="60" units="seconds" value="30">
+			<Help>seconds; after successful code entry and unit unlocks, it will automatically re-lock after specified time (30 = default value)</Help>
+		</Value>
+		<Value type="byte" genre="config" instance="1" index="3" label="Auto-re-lock Time" size="1" min="10" max="90" units="seconds" value="10">
```
Comment (zxdavb) [inline config/assa_abloy/KeylessConnected-plus.xml]: I think this should be `index="6"` and not `index="3"`:
```
<Value type="byte" genre="config" instance="1" index="6" label="Auto-re-lock Time" size="1" min="10" max="90" units="seconds" value="10">
```

File: config/manufacturer_specific.xml [hunk 2]
```diff
@@ -217,6 +217,7 @@
 		<Product type="0007" id="0000" name="Yale Keyless Connected Smart Lock (YSL)" config="assa_abloy/KeyfreeConnected.xml"/>
 		<Product type="1bef" id="9230" name="Yale Keyless Connected Smart Lock (YSL)" config="assa_abloy/KeyfreeConnected.xml"/>
 		<Product type="6600" id="0002" name="Yale Conexis L1 (SD-L1000-CH)" config="assa_abloy/ConexisL1.xml"/>
+		<Product type="0007" id="0001" name="Keyless Connected (YD-01-CON)" config="assa_abloy/KeylessConnected-plus.xml"/>
```
Comment (zxdavb) [inline config/manufacturer_specific.xml]: I think `YD-01-CON` and `YKFCON` can share the same config file:
```
<Product type="0007" id="0001" name="Keyless Connected (YD-01-CON)" config="assa_abloy/KeyfreeConnected-plus.xml"/>
```

**Comment by nechry** [comment]
Thanks

**Comment by Dinth** [comment]
Small OT: I had Keyfree lock for ages and it always worked with OZW (HomeAssistant) perfectly fine. 
Assa Abloy
product name
Unknown: type=0600, id=0000
I didnt know that those locks need some special support from OZW. Is there any functionality im missing in my lock?

**Pull request merged**